src/main/java/org/tron/core/net/node/NodeDelegateImpl.java

```diff
@@ -172,7 +172,7 @@ public boolean handleTransaction(TransactionCapsule trx) throws BadTransactionEx
       logger.warn(e.getMessage());
       throw new BadTransactionException();
     } catch (TooBigTransactionResultException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this "catch" keyword to a new dedicated line. ([squid:RightCurlyBraceDifferentLineAsNextBlockCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1500&issues=AWXBxdl6ashydt7ke7Ei&open=AWXBxdl6ashydt7ke7Ei)

[ISSUE_KEY]: AWXBxdl6ashydt7ke7Ei